### PR TITLE
figure edits for review

### PR DESCRIPTION
**Pull request by jclagarde**
Screen Shot 2021-10-26 at 5 48 54 PM
Screen Shot 2021-10-20 at 5 50 01 PM

So for line 108 and 112 for figure4.py, I was getting an error that had to do with the spelling of `censore` I had changed it and it resolved that specific issue but it led to more errors 

Screen Shot 2021-10-26 at 5 52 40 PM
My main confusion is this error because `cell.left` is defined already within the CellVar class

### CODE REVIEW

**Comment by Farnazmdi** [comment]
I hadn't seen your message here, JC.
I don't think anything has changed. I suggest pick the best one you see among those.

**Comment by bmesbuildteamucla** [comment]
> I hadn't seen your message here, JC. I don't think anything has changed. I suggest pick the best one you see among those.

I'm experiencing similar merge issues as the last time where the easiest solution was to just create a new branch. Since I didn't edit too many lines of code, I'll just go ahead and copy these changes onto a new branch

**Comment by jclagarde** [comment]
> I hadn't seen your message here, JC. I don't think anything has changed. I suggest pick the best one you see among those.

I'm experiencing similar merge issues as the last time where the easiest solution was to just create a new branch. Since I didn't edit too many lines of code, I'll just go ahead and copy these changes onto a new branch

**Comment by jclagarde** [comment]
I was able to resolve the merge issue without creating a new branch and I remade a figure 4 with a larger number of cells. Please take a look at it when you have the chance and let me know if I should make any further edits. Thank you! :)

**Comment by Farnazmdi** [comment]
Thanks JC, figure 4 looks great now.

File: lineage/plotTree.py [hunk 1]
```diff
@@ -5,11 +5,11 @@
 import networkx as nx
 
 cs = ['lightblue', 'orange', 'lightgreen', 'red', 'purple', 'olive', 'gray']
-stateColors = ['blue', 'orange', 'green', 'red', 'purple', 'olive', 'gray']
+stateColors = ['blue', 'green', 'orange', 'red', 'purple', 'olive', 'gray']
```
Comment (Farnazmdi) [inline lineage/plotTree.py:8]: Could you return this back to the original now that you created figure?

**Review by Farnazmdi — APPROVED** [review]
Other than one tiny edit, looks good.Thanks

**Review by Farnazmdi — COMMENTED** [review]
Wait,
In figureS8-S10 the y_label and title are the same. THey should not be.
Please edit this.

image

File: lineage/plotTree.py (hunk 1 above)
Comment (jclagarde) [reply lineage/plotTree.py]: > Wait, In figureS8-S10 the y_label and title are the same. THey should not be. Please edit this.
> 
> image

Sure thing. Just confirming, the y-axis label should be "Accuracy [%]" , correct?

**Comment by jclagarde** [comment]
> Wait, In figureS8-S10 the y_label and title are the same. THey should not be. Please edit this.
> 
> image

also now that I look at it, should I also remove the light blue and light green lines?

**Comment by Farnazmdi** [comment]
Those are the true values, should be there for comparison. But you can limit the yaxis limits of (a) and (e) from 0-1 to 0.5-1, and 0-15 for (f), (g).

**Comment by jclagarde** [comment]
> Those are the true values, should be there for comparison. But you can limit the yaxis limits of (a) and (e) from 0-1 to 0.5-1, and 0-15 for (c), (d), (f), (g).

Just edited the y-axis limits and labels. Please let me know if this is what you had in mind when you get the chance

**Comment by Farnazmdi** [comment]
Yes thank you. I will have to close this PR as I integrated your edits into my PR.
Great work, thank you.

**Comment by jclagarde** [comment]
> Yes thank you. I will have to close this PR as I integrated your edits into my PR. Great work, thank you.

Sounds good, thank you!

**Pull request closed without merging**